### DISCUSSION TITLE
Web search with open-webui

### DISCUSSION THREAD

**Discussion by taidbui** [open-webui/open-webui#971] [General]
Given my enjoyment of using the Open Webui for running local LLMs with RAG, I am curious if web search is being considered in the development roadmap. If it happens, it will be a really big shot tbh!

**Comment by tjbck** [comment]
Related: #586

**Reply by taidbui** [reply]
@tjbck: I am aware of the RAG with a particular webpage. What I meant here is proper web search similar to https://www.perplexity.ai do. Please feel free to reopen this if this makes sense.

**Reply by tjbck** [reply]
That's the plan with #586 :)

**Reply by taidbui** [reply]
Great. Thanks.
